doctrine/doctrine-bundle/1.6/manifest.json

```diff
@@ -10,7 +10,29 @@
         "#1": "Format described at https://www.doctrine-project.org/projects/doctrine-dbal/en/latest/reference/configuration.html#connecting-using-a-url",
         "#2": "For an SQLite database, use: \"sqlite:///%kernel.project_dir%/var/data.db\"",
         "#3": "For a PostgreSQL database, use: \"postgresql://db_user:db_password@127.0.0.1:5432/db_name?serverVersion=11&charset=utf8\"",
-        "#4": "IMPORTANT: You MUST configure your server version, either here or in config/packages/doctrine.yaml",
+        "#4": "For the Docker integration, use: \"mysql://symfony:ChangeMe@db:3306/app?serverVersion=mariadb-10.5.6\"",
+        "#5": "IMPORTANT: You MUST configure your server version, either here or in config/packages/doctrine.yaml",
         "DATABASE_URL": "mysql://db_user:db_password@127.0.0.1:3306/db_name?serverVersion=5.7"
+    },
+    "dockerfile": ["RUN docker-php-ext-install pdo_mysql"],
+    "docker-compose": {
+        "services": [
+            "db:",
+            "  image: mariadb:10.5.6",
+            "  environment:",
+            "    - MYSQL_DATABASE=app",
+            "    # You should definitely change the password in production",
+            "    - MYSQL_PASSWORD=ChangeMe",
+            "    - MYSQL_RANDOM_ROOT_PASSWORD=true",
+            "    - MYSQL_USER=symfony",
+            "  volumes:",
+            "    - db-data:/var/lib/mysql:rw",
+            "    # You may use a bind-mounted host directory instead, so that it is harder to accidentally remove the volume and lose all your data!",
+            "    # - ./docker/db/data:/var/lib/mysql:rw",
+            "  # To expose the port publicly",
+            "  # ports:",
+            "  #   - \"3306\""
```

Comment:
It's hardcode in Flex : https://github.com/symfony/flex/blob/main/src/Configurator/DockerComposeConfigurator.php#L34-L35

Reply:
Well, it's probably a bit too much, but I'd recommend the creation of a special configurator for a `docker-compose.override.yml.dist` file, so maintainers can explicitly define what can be "publicly" exposed, or what kind of "machine-specific" configuration can be used. 

Exposing public ports is not the best, it can cause conflicts, or priority issues (like two apps listening to the same port), etc., and having them directly in the `docker-compose.yml` file, even commented, might invite users to put the public ports in it.

Having the `dist` file invites the user to create their own use-case when needed, instead of having it enabled by default.

Reply:
> Well, it's probably a bit too much, but I'd recommend the creation of a special configurator for a docker-compose.override.yml.dist file, so maintainers can explicitly define what can be "publicly" exposed, or what kind of "machine-specific" configuration can be used.

@Pierstoval haha, actually I'm already on it. But it's not a blocker for this one. We'll be able to update it to leverage the new capabilities of the configurator later.

> Exposing public ports is not the best, it can cause conflicts, or priority issues (like two apps listening to the same port), etc., and having them directly in the docker-compose.yml file, even commented, might invite users to put the public ports in it.

It will be not be necessary anymore with the new configurator, indeed.

Reply:
> actually I'm already on it

You're always in advance 🤣 👍

Reply:
And here we are: symfony/flex#705

Reply:
I don't understand the issue here. On the localhost, the port is going to be random, so there is no possible conflicts.

Reply:
If we put the port in `docker-compose.yml` instead of not in `docker-compose.override.yml` the port will be exposed publicly even in production, which can be a security issue.